supporting/shimexe/shim.cs

```diff
@@ -91,6 +91,12 @@ internal struct PROCESS_INFORMATION {
             if(!string.IsNullOrEmpty(cmd_args)) cmd_args = " " + cmd_args;
             var cmd = "\"" + path + "\"" + cmd_args;
 
+            foreach (KeyValuePair<string, string> entry in config) {
+                if (entry.Key.StartsWith("env::")) {
```

Comment:
Use similar syntax to PowerShell `$env:PATH` > `env:PATH`. Easier to copy and paste :smile:
```suggestion
                if (entry.Key.StartsWith("env:")) {
```

Reply:
Okay!